### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by Cyrus-Kiprop**
This is a weather app built using vanilla javascript, HTML, and CSS.  The user can search and request weather data for cities. Weather data consist of wind, humidity, temperature, and general feel of the day.
Requirements
1. User can toggle between Fahrenheit and Celsius units
2. Asynchronous fetching of data using promises/async and await

### CODE REVIEW

File: src/components/fetchData.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    .then((result) => Db(result))
+    .catch(() => {
+      const wrapper = document.getElementsByClassName(
+        'weather__app-wrapper',
+      )[0];
+      wrapper.appendChild(cityNotFound());
+    });
+}
+
+export default fetchDetails;
```
Comment (shloch) [inline src/components/fetchData.js]: this is like the core of of your app, you need to separate LOGIC from DOM
That's all you need to do [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) ]

**Review by shloch — CHANGES_REQUESTED** [review]
weather

Great job.............. well built

Please separate DOM from LOGIC, that's all you need to do

File: src/components/container.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import utils from './UxUtils';
+import weatherAppWrapper from './weatherDetailsWrapper';
+
+const { make } = utils;
+
+const container = make('div', 'app-wrapper');
+
+// utils.appendBulkchild(container, [weatherAppWrapper]);
```
Comment (shshamim63) [inline src/components/container.js:8]: Please remove the commented code.

File: src/components/stats/cityForm.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    id: 'search-city',
+    placeholder: 'Enter a city...',
+  });
+
+  const btn = make('button', 'btn', form, {
+    type: 'submit',
+  });
+  make('i', 'fa fa-search', btn);
+
+  // btn.appendChild(document.createTextNode('Submit'));
```
Comment (shshamim63) [inline src/components/stats/cityForm.js:29]: Please remove it too.

**Review by shshamim63 — APPROVED** [review]
## :100: Approved :+1: 
___
Hi @Cyrus-Kiprop :wave: 
Personally I loved the view of your page. And the background picture looks amazing :sunglasses: .Just found some commented codes. Your project should not contain any commented code. After removing them you are good to advance. This :+1: is from me.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Pull request merged**